week1_session4_intro_to_ml_and_scikit_learn.ipynb

```diff
@@ -0,0 +1,1547 @@
+{
```

Comment:
<p>Maybe save results to <code>results/week1_session4_scikit-learn/</code>  instead of <code>images</code>  to make it easier to link results to the respective notebook? This we could streamline with other notebooks' results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/ai_in_medicine/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>dones :)</p>